components/kyma-environment-broker/internal/cls/provisioner.go

```diff
@@ -42,57 +40,54 @@ type ProvisionRequest struct {
 }
 
 type ProvisionResult struct {
-	InstanceID            string
-	ProvisioningTriggered bool
-	Region                string
+	InstanceID string
+	Region     string
 }
 
-func (p *provisioner) Provision(smClient servicemanager.Client, request *ProvisionRequest) (*ProvisionResult, error) {
+func (p *provisioner) Provision(log logrus.FieldLogger, smClient servicemanager.Client, request *ProvisionRequest) (*ProvisionResult, error) {
```

Comment:
the same like for deprovision - log at the end

Reply:
missed it. Fixed now.